tests/e2e_tests/e2e_test_config.yml

```diff
@@ -5,6 +5,8 @@ DB:
   TYPE: json_file
   PATH: tests/e2e_tests/e2e_test_data.json
 
+FEATURE_FLAGS:
+
```

Comment:
```suggestion
FEATURE_FLAGS:
```suggestion
```
```

Reply:
What is this? This isn't a valid yaml syntax, I get an error on yaml scanning